overwatch/src/overwatch/mod.rs

```diff
@@ -117,6 +157,11 @@ pub trait Services: Sized {
     /// Update service settings for all services attached to the trait
     /// implementer.
     fn update_settings(&mut self, settings: Self::Settings);
+
+    /// Get the [`LifecycleHandle`] for a service attached to the trait
+    /// implementer.
+    fn get_service_lifecycle_handle(&self, service_id: &Self::RuntimeServiceId)
```

Comment:
Should this be exposed in the trait? I feel this should rather be consumed as part of the lifecycle operations like `start` and `stop`, no?

Reply:
Yup, you're right. Again, not sure what I was looking at then 😄